src/profiler/server.ts

```diff
@@ -0,0 +1,251 @@
+import * as inspector from 'inspector';
+import { once } from 'events';
+import { Server } from 'http';
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import { startProfiler, stopProfiler } from 'stacks-encoding-native-js';
```

Comment:
FYI I think this will generally only show useful data for invocations into code exposed by this rust library

Reply:
Do you think we should split it into a separate tool?

Reply:
Yeah I think `stacks-encoding-native-js` is a pretty heavy dependency to add to this library

Reply:
Ok will do, perhaps it would make sense to add this native profiler to that library directly instead (as an option for debugging)